crates/bevy_pbr/Cargo.toml

```diff
@@ -51,6 +51,7 @@ bincode = "1"
 range-alloc = "0.1"
 nonmax = "0.5"
 static_assertions = "1"
+wgpu = { version = "0.19.3", default-features = false, features = [] }
```

Comment:
Can this be avoided somehow? This is now necessitated by having to import `wgpu::DownlevelFlags`.

Reply:
We should re-export the required type, probably in `bevy_render`.

Reply:
Right! That makes sense. This should be fixed now.